.gitignore

```diff
@@ -38,3 +38,13 @@ packages/*/build
 # This prevents accidental commit of this token to the repository.
 nx-cloud.env
 .nx
+
+# Yarn Modern
+# @see: https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored
+.pnp.*
```

Comment:
Is this also related to Yarn?

Reply:
I have used this setting from the documentation (https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored). Since we are not using zero-install we can safely ignore this.